pkg/controller/flowsconfig/flowsconfig_controller.go

```diff
@@ -0,0 +1,135 @@
+package flowsconfig
```

Comment:
I don't think this is 100% correct implementation wise... If you were trying to make a new controller, it would need to follow the same schema all the other controllers follow.. 

For example, the `egress_router_controller` defines an `Add` function which is added to the list of controller create functions [here](https://github.com/openshift/cluster-network-operator/blob/master/pkg/controller/add_networkconfig.go#L16)

Really all we're doing here is providing another input to the `operconfig` controller

Reply:
I would probably move these files/ helper functions to the `operconfig/` directory

Reply:
In the previous commit I needed to invoke it from the operconfig controller, instead of creating a new one, because the OVN-K configuration is rendered together with the rest of the network configuration (which is invoked from the operconfig).

My initial decision of having everything in a package apart was to put together the bootstrapping/rendering and the reconciliation of the ConfigMap option, but it's true that could give the false impression of being a standalone controller. In the following commit I'll do:

- move the flowsconfig bootstrap function to the `ovn_kubernetes.go` file, to avoid import cycles.
- move watch and reconcile to the `operconfig` package.